txs/tx.go

```diff
@@ -114,11 +114,6 @@ type (
 		DecodeTx(txBytes []byte) (Tx, error)
 	}
 
-	// UnconfirmedTxs
```

Comment:
assuming since the endpoint is still available that this information can still be pulled? which has been helpful at times in debugging whether a mempool is queueing...

Reply:
Oh yes it is this, but this in an RPC view model and just didn't belong here - it was unused since refactor but noticed it during these changes.

Reply:
👍